airflow/providers/amazon/provider.yaml

```
@@ -89,6 +89,7 @@ versions:
 
 dependencies:
   - apache-airflow>=2.7.0
+  - apache-airflow-providers-common-compat
```

```suggestion
  - apache-airflow-providers-common-compat>=1.1.0
```
everywhere, and common.compat version should also be bumpted to 1.1.0

---

FYI, you might bump into the same CI error as Maciej did: https://github.com/apache/airflow/pull/40819#discussion_r1684277036

---

I think I added it in all proper places. Also followed advice how to avoid CI error from Kacper's link.